clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1581,12 +1581,16 @@ protected void checkIngestionProgress(Store store) throws InterruptedException {
   }
 
   protected void updateIngestionRoleIfStoreChanged(Store store) throws InterruptedException {
```

Comment:
We should rename `updateIngestionRoleIfStoreChanged` since it doesn’t accurately reflect what the method does. The ingestion role isn’t being changed directly. Instead, the method refreshes the ingestion context when necessary. A more suitable name could be `refreshIngestionContextIfChanged` or please use something that better captures its purpose.

Reply:
Yeah, refresh is better term.